sql/core/src/main/scala/org/apache/spark/sql/sources/ddl.scala

```diff
@@ -50,6 +50,7 @@ private[sql] class DDLParser extends StandardTokenParsers with PackratParsers wi
     lexical.allCaseVersions(k.str).map(x => x : Parser[String]).reduce(_ | _)
 
   protected val CREATE = Keyword("CREATE")
+  protected val DECIMAL = Keyword("DECIMAL")
```

Comment:
Oh, actually, I put this comment on the wrong line. I meant to put it under the line 108.